plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -297,8 +439,21 @@ export default class NodesTable extends React.Component<
               />
             }
             cellRenderer={memRenderer}
-            growToFill={true}
-            minWidth={110}
+            growToFill={
```

Comment:
This is all fine for this PR, but I wonder if there could be a future improvement to how we handle these resizeable columns...  Having the Store to store this state makes a lot of sense so having this kind of logic in dcos-ui vs ui-kit is reasonable.  Alot of this PR, esp the lines following, are boilerplate-ish and some of potentially error-proned, like this growToFill "logic"...  Would there be a slick way to have a HOC or some other kind of component/helper in dcos-ui that can almost "generate" this boilerplate to make the ultimate api something like:

<Column minWitdth={110} resizeable={true} columnKey={"mem"} />

Reply:
I'd agree with Jackson. We should design the interface a little here.

```jsx
<ConfigurableTable id="table_id" storeBackend={LocalStorageBackend}>
    <ResizableColumn minWitdth={110} columnKey="mem" />
</ConfigurableTable>
```

or even

```jsx
<Configuration id="table_id" storeBackend={LocalStorageBackend}>
  <Table>
    <ResizableColumn minWitdth={110} columnKey="mem" />
  </Table>
</Configuration>
```

Configuration would inject defaults into its child component. We would need figuring out the `onConfigurationChange` interface though.